### PR TITLE
Docparser rewrite

### PR DESCRIPTION
**Pull request by kantenkugel**
Rewrite of DocParser.
Now parses the generated html javadocs
Also uses embeds for pretty display

### CODE REVIEW

**Comment by kantenkugel** [comment]
You might want to Squash-merge

File: config.template.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "discord": {
+    "token": "MTUwMjAzODQxODI3MDQ1Mzc2.DBQVvw.7V1ZiVyqBmbzZzrNKEJy4PaBTf8"
```
Comment (Almighty-Alpaca) [inline config.template.json]: 😆

Comment (kantenkugel) [reply config.template.json]: Yea... I think I had the token reset about a min after push 😆

**Comment by kantenkugel** [comment]
Should be ready

**Review by Almighty-Alpaca — APPROVED** [review]

**Pull request merged**